.github/workflows/CI.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           sudo apt install -y libmpfr-dev
           sudo apt install -y autoconf
           sudo apt install -y libtool-bin
+          sudo apt install -y gcovr
```

Comment:
Where are you using gcovr? We used to use it in GAP but not anymore

Reply:
Because I couldn't find out how to install `gcov` on Ubuntu, perhaps it is included with GCC.